infrastructure/terraform/02-inputs-api-management.tf

```diff
@@ -24,4 +24,7 @@ variable "open_api_spec_content_value" {
 }
 variable "service_url" {
   description = "Absolute URL of the backend service implementing this API."
+}
```

Comment:
same as above comment, this change is already should be your base, nothing relevant to the unit tests